evalml/tests/model_understanding_tests/prediction_explanations_tests/test_algorithms.py

```diff
@@ -70,7 +72,10 @@ def test_value_errors_raised(mock_tree_explainer, pipeline, exception, match):
         pytest.importorskip("catboost", "Skipping test because catboost is not installed.")
 
     with pytest.raises(exception, match=match):
-        _ = _compute_shap_values(pipeline({}), pd.DataFrame(np.random.random((2, 16))))
+        if pipeline == TimeSeriesBaselineRegressionPipeline:
```

Comment:
Thanks for updating this test! Up to you but I would prefer to skip the if/else since you can specify pipeline parameters for any pipeline class:

```python
    with pytest.raises(exception, match=match):
        _ = _compute_shap_values(pipeline({"pipeline": {"gap": 0, "max_delay": 0}}), pd.DataFrame(np.random.random((2, 16))))
```